src/components/common/YourCompany/ChangeBusinessType.vue

```diff
@@ -348,6 +374,10 @@ export default class ChangeBusinessType extends Mixins(CommonMixin) {
   @Watch('selectedEntityType')
   private clearConfirmArticles (): void {
     this.confirmArticles = false
+    // Ensure when selected type changes by NR, show the error message and Articles
+    if (this.selectedEntityType !== this.getOriginalLegalType) {
+      this.isEditingType = true
```

Comment:
This is to fix the bug when click "Change" in name change section: enter a new NR that changes the business to a different type, the confirmArticles and error msg won't show.
The if condition is to avoid expanding this section when the component is initially loaded

Reply:
I love the comments explaining why this code exists.